_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,310 @@
+    `-blockfilterindex` configuration parameter can now respond to `getcfcheckpt`,
+    `getcfheaders`, and `getcfilters` requests with the corresponding `cfcheckpt`,
+    `cfheaders`, and `cfilters` responses. The node does not yet advertise support
+    for BIP157 with `NODE_COMPACT_FILTERS` in its version message. The final step,
+    [Bitcoin Core #19070][], is under review at the time of this writing and
+    would enable nodes to signal the ability to serve compact block filters.
+    The feature is disabled by default and can be enabled with the
+    `-peerblockfilters` configuration parameter.
+
+- [Bitcoin Core #16939][] changes how long Bitcoin Core waits until
+  it queries DNS seeds for the IP addresses of potential peers.
+  Previously, if the node had peer IP addresses in its database, it
+  would try opening several connections and wait 11 seconds for
+  successful connections before requesting new addresses.  Now, if it
+  has more than 1,000 IP addresses in its database---which is common for
+  nodes that have been online more than a few hours---it'll wait up to 5
+  minutes before querying.  This improves the chance that a restarted
+  node will entirely use P2P address discovery without relying on the
+  centralized DNS seeds.
+
+- [LND #4228][] adds a new wallet command `labeltx` for labelling past onchain
+  transactions. This is a continuation of the work done in [LND #4213][] which
+  allowed setting a label when sending a payment.  Labels are personal
+  wallet metadata that help the user remember who they paid and what
+  they bought; the labels aren't part of the onchain transactions and
+  aren't shared with any other user.
+
+## On the occasion of Optech Newsletter #100
+
+> "I am somewhat surprised that nobody has taken to [writing] weekly
+> summaries of research and development activity. Summarizing recent
+> work is a valuable task that others can engage in just by reading the
+> mailing list and aggregating multiple thoughts together."
+>
+> {:.right}
+> ---Bryan Bishop, [19 August 2015][bishop summaries]
+
+{% comment %} <!--
+  $ find _posts/en/newsletters/ _includes/specials/bech32/  -type f | xargs cat | wc -w
+  155217
+
+  443 pages = 155217 words / 350 words per printed page
+-->{% endcomment %}
+
+Almost five years after Bishop made the above comment, we remain
+convinced that writing weekly summaries of research and development
+activity is a task that's valuable to both the open source Bitcoin
+development community and to the many businesses that depend upon the
+community's work.  But in the two years we've been producing this
+newsletter, we've also discovered that summarizing isn't quite as quick
+and simple as we initially expected it to be.  Accordingly, we'd like to
+take this chance to thank the people who make this newsletter possible
+by generously contributing a significant amount of their valuable time
+week after week: [Adam Jonas][], [Carl Dong][], [David A. Harding][],
```

Comment:
I have to say I'm uncomfortable being mentioned first on this line with @harding buried in the middle (I get it's alphabetical). I don't believe that credit should be equally shared. Can emphases or an additional sentence be added to make it clear who does most of the work to make this newsletter happen?

Reply:
I'm sorry this makes you feel uncomfortable.  :-(  However, an alphabetical list of contributors in the standard for lists of credits in most free software projects even though all of those projects have the same situation of different people contributing different amounts of time to the project.  I think we're better off following that simple and pragmatic standard than we are trying to fairly apportion credit among ourselves.

Reply:
@harding Just feel that you deserved to be highlighted since you put in the most work.